stable/testlink/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: testlink
-version: 7.0.3
+version: 7.0.4
```

Comment:
We're adding a new feature, therefore we need to bump the minor version

Reply:
done